.pre-commit-config.yaml

```diff
@@ -27,8 +27,20 @@ repos:
         files: "gym-unity/.*"
         args: [--ignore-missing-imports, --disallow-incomplete-defs]
 
+-   repo: https://gitlab.com/pycqa/flake8
```

Comment:
As of https://github.com/pre-commit/pre-commit-hooks v2.5.0, their flake8 hook is deprecated: https://github.com/pre-commit/pre-commit-hooks/releases/tag/v2.5.0